api/src/main/webapp/WEB-INF/cron_perf.yaml

```diff
@@ -0,0 +1 @@
+cron:
```

Comment:
RE separate PR: I think I'd prefer to leave it in, since this PR doesn't really do anything without at least one environment override.

RE cron disablement design: I filed a ticket and put my rationale in there, PTAL.

Reply:
I'm backtracking and going with your original suggestion to split out the perf change. It's Friday and I don't know that we've established we definitely want crons off in perf